baseapp/baseapp.go

```diff
@@ -111,7 +111,7 @@ type BaseApp struct { // nolint: maligned
 	// ResponseCommit.RetainHeight.
 	minRetainBlocks uint64
 
-	// application's version string
+	// version represents the application software semantic version.
 	version string
```

Comment:
Do we need a separate field? IMO we only need the AppVersion field

Reply:
version is used to represent the applications semantic software version, tendermint asks for both. In most cases I think its empty because it set via a flag.